packages/react-native-client/src/hooks/useFishjamEvent.ts

```diff
@@ -13,6 +13,7 @@ export const ReceivableEvents = {
   ReconnectionRetriesLimitReached: 'ReconnectionRetriesLimitReached',
   ReconnectionStarted: 'ReconnectionStarted',
   Reconnected: 'Reconnected',
```

Comment:
Can these values be removed now?

Reply:
It depends. Do we want to remove those events for users completely? I left them in case someone relies on them and doesn't want to switch to reconnection status